src/clipboard-action-default.js

```diff
@@ -34,98 +35,20 @@ class ClipboardAction {
    */
   initSelection() {
     if (this.text) {
-      this.selectFake();
+      this.selectedText = ClipboardActionCopy(this.text, {
+        container: this.container,
+      });
     } else if (this.target) {
-      this.selectTarget();
-    }
-  }
-
-  /**
-   * Creates a fake textarea element, sets its value from `text` property,
-   */
-  createFakeElement() {
-    const isRTL = document.documentElement.getAttribute('dir') === 'rtl';
-
-    this.fakeElem = document.createElement('textarea');
-    // Prevent zooming on iOS
-    this.fakeElem.style.fontSize = '12pt';
-    // Reset box model
-    this.fakeElem.style.border = '0';
-    this.fakeElem.style.padding = '0';
-    this.fakeElem.style.margin = '0';
-    // Move element out of screen horizontally
-    this.fakeElem.style.position = 'absolute';
-    this.fakeElem.style[isRTL ? 'right' : 'left'] = '-9999px';
-    // Move element to the same position vertically
-    let yPosition = window.pageYOffset || document.documentElement.scrollTop;
-    this.fakeElem.style.top = `${yPosition}px`;
-
-    this.fakeElem.setAttribute('readonly', '');
-    this.fakeElem.value = this.text;
-
-    return this.fakeElem;
-  }
-
-  /**
-   * Get's the value of fakeElem,
-   * and makes a selection on it.
-   */
-  selectFake() {
-    const fakeElem = this.createFakeElement();
-
-    this.fakeHandlerCallback = () => this.removeFake();
-
-    this.fakeHandler =
-      this.container.addEventListener('click', this.fakeHandlerCallback) ||
-      true;
-
-    this.container.appendChild(fakeElem);
-
-    this.selectedText = select(fakeElem);
-
-    this.copyText();
-
-    this.removeFake();
-  }
-
-  /**
-   * Only removes the fake element after another click event, that way
-   * a user can hit `Ctrl+C` to copy because selection still exists.
-   */
-  removeFake() {
-    if (this.fakeHandler) {
-      this.container.removeEventListener('click', this.fakeHandlerCallback);
-      this.fakeHandler = null;
-      this.fakeHandlerCallback = null;
-    }
-
-    if (this.fakeElem) {
-      this.container.removeChild(this.fakeElem);
-      this.fakeElem = null;
-    }
-  }
-
-  /**
-   * Selects the content from element passed on `target` property.
-   */
-  selectTarget() {
-    this.selectedText = select(this.target);
-    this.copyText();
-  }
-
-  /**
-   * Executes the copy operation based on the current selection.
-   */
-  copyText() {
-    let succeeded;
-
-    try {
-      succeeded = document.execCommand(this.action);
-    } catch (err) {
-      succeeded = false;
+      if (this.action === 'cut') {
+        this.selectedText = ClipboardActionCut(this.target);
+      } else if (this.action === 'copy') {
+        this.selectedText = ClipboardActionCopy(this.target, {
+          container: this.container,
+        });
```

Comment:
I think in this case you can do something like:
```js
this.selectedText = this.action === 'cut' ? ClipboardActionCut(this.target) : ClipboardActionCopy(this.target, { container: this.container });
```

What do you think about it?